pkg/api/v1/validation/validation.go

```diff
@@ -70,12 +70,16 @@ func ValidateResourceRequirements(requirements *v1.ResourceRequirements, fldPath
 
 func validateContainerResourceName(value string, fldPath *field.Path) field.ErrorList {
 	allErrs := validateResourceName(value, fldPath)
-	if len(strings.Split(value, "/")) == 1 {
-		if !helper.IsStandardContainerResourceName(value) {
-			return append(allErrs, field.Invalid(fldPath, value, "must be a standard resource for containers"))
-		}
+
+	// Container resources must be one of:
+	// - A known resource in the `*kubernetes.io/` namespace.
+	// - An extended resource (not in the `*kubernetes.io/` namespace.
+	// - An opaque integer resource [DEPRECATED] (TODO: Remove following deprecation)
+	if !helper.IsStandardContainerResourceName(value) &&
```

Comment:
Can we do a canvas (GitHub search / dev list email) about people abusing kubernetes.io/* in the wild?  If this just restricts only kubernetes.io AND we warned about it AND we are convinced we won't break someone important horribly I think we can allow it.